SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier2/PoisonousGland.cs

```diff
@@ -143,11 +159,11 @@ public void ModifyStatArguments(RecalculateStatsAPI.StatHookEventArgs args)
             }
             private void UpdateDebuffs()
             {
-                if (buffCount == 0) return;
+                if (base.gameObject == null || buffCount == 0) return;
```

Comment:
this is indirection, change to
```suggestion
                if (this == null || buffCount == 0) return;
```